Sources/XCTest/Public/Asynchronous/XCTestExpectation.swift

```diff
@@ -11,7 +11,7 @@
 //
 
 /// Expectations represent specific conditions in asynchronous testing.
-open class XCTestExpectation {
+open class XCTestExpectation: @unchecked Sendable {
```

Comment:
This is actually dangerous, because the class is `open` and subclasses could capture data that is _not_ sendable. We should only make instances sendable when they actually _are_.

This is an unfortunate problem with the current design of the class: there's no distinction between the base class `open XCTestExpectation` and some concrete `final XCTDescribedExpectation` class that takes a description and can't possibly become unsendable.

I don't think that's solvable in the general case, and XCTest as it ships with Xcode has this same problem.